src/main/scala/com/cloudera/sparkts/Frequency.scala

```diff
@@ -100,8 +100,9 @@ class DayFrequency(val days: Int)
   extends PeriodFrequency(ChronoUnit.DAYS.getDuration.multipliedBy(days)) {
 
   override def difference(dt1: ZonedDateTime, dt2: ZonedDateTime): Int = {
-    val period = Period.between(dt1.toLocalDate, dt2.toLocalDate)
-    period.getDays / days
+    //val period = Period.between(dt1.toLocalDate, dt2.toLocalDate)
```

Comment:
Yes. It is intended to be a temp fix to pass the builds.
A complete fix should be considered for #87